elastic/integrations
# IIS Integration - pipeline update for the log format

## Type of change
- Enhancement

## What does this PR do?

Fixes the issue reported in #2347 . Supports the given log format

`2021-11-28 00:00:04 W3SVC1 10.44.0.136 GET /cdgo SERVER-STATUS=200 443 - 67.43.156.13 HTTP/1.1 Chrome/2.12.1 - example.com 200 0 0 34018 303 46`

## Checklist



## How to test this PR locally

The new log format is added to the test-iis-access.log
Execute the pipeline test to verify the given log format is working.

## Related issues

- Closes #2347
